test/test_query2.cpp

```diff
@@ -5225,6 +5239,13 @@ TEST(Query_LinksTo)
     q = source->column<Link>(col_linklist) != target->get_object(target_keys[6]);
     found_key = q.find();
     CHECK_EQUAL(found_key, source_keys[2]);
+
+    q = source->where().equal(col_linklist, Mixed(target_keys[0]));
+    tv = q.find_all();
+    CHECK_EQUAL(tv.size(), 2);
+    q = source->where().not_equal(col_linklist, Mixed(target_keys[6]));
+    tv = q.find_all();
+    CHECK_EQUAL(tv.size(), 2);
```

Comment:
Could you add some checks for searching for == and != null, across linklist and link? Just making sure we haven't changed behaviour from before.

Reply:
Well - this is new functionality. We could not do this before. LinksToNode would never match on a null link. But I realize that you would expect to match null links on a link?.

Reply:
Right, good point. Thanks for adding tests anyhow 😄